src/sage/libs/pari/__init__.py

```diff
@@ -170,8 +170,33 @@
     3.605463601432652085915820564207726774810268996598024745444380641429820491740 # 64-bit
     3.60546360143265208591582056420772677481026899659802474544                    # 32-bit
 
+
+    Bernoulli Number Function
+    -------------------------
+
+    The Bernoulli numbers Bn are a sequence of rational numbers which occur frequently in analysis.
+    PARI algorithm to computes the Bernoulli numbers Bn, for n >= 0, by using Bn=zeta(n)*(2*n!)/(2^n pi^n).
```

Comment:
You should convert your formulas to valid latex and mark it as such. This is one example:
```suggestion
    PARI algorithm to computes the Bernoulli numbers `B_n`,
    for `n \geq 0`, by using `B_n = \zeta(n) \frac{2 n!}{2^n \pi^n}`.
```